src/test/java/uk/gov/companieshouse/overseasentitiesapi/model/DtoDaoMappingTest.java

```diff
@@ -167,7 +168,7 @@ private OverseasEntitySubmissionDto getOverseasEntitySubmissionDto() {
 
         EntityDto entity = new EntityDto();
         entity.setName("name");
-        entity.setEmail("email");
+        entity.setEmail(" " + EMAIL + " ");
```

Comment:
maybe add comment on line above to highlight the the spaces are used to test the trim functionality (just in case in future someone changes it without realising)

Reply:
or create a EMAIL_NEEDS_TRIMMING constant (or better named)

Reply:
Commented added but I've also defined a constant with a more meaningful name (with Javadoc comment).